rfml.go

```diff
@@ -663,7 +663,13 @@ func sanitizeTestTitle(title string) string {
 
 	// replace all non-alphanumeric characters with an underscore
 	rep := regexp.MustCompile(`[^[[:alnum:]]`)
-	return rep.ReplaceAllLiteralString(title, "_")
+	title = rep.ReplaceAllLiteralString(title, "_")
+
+	if len(title) > 16 {
+		return title[:16]
+	}
```

Comment:
It does because if the length is less than 16, you access parts of the slice that are out of bounds. I made the same assumption when writing this myself 😅